### PR TITLE
Latex enabled if necessay and title list handled properly

### PR DESCRIPTION
**Pull request by areeda**
Addresses issue #975 and reinstates latex when needed

### CODE REVIEW

File: gwpy/cli/cliproduct.py [hunk 1]
```diff
@@ -639,18 +639,20 @@ def set_legend(self):
...
+
         if title is None:
-            title = self.get_title().rstrip(', ')
-        if self.usetex:
-            title = label_to_latex(title)
-        if title:
-            self.ax.set_title(title, fontsize=12)
-            self.log(3, ('Title is: %s' % title))
+            title = [self.get_title().rstrip(', ')]
+        for title_line in title:
```
Comment (duncanmmacleod) [inline gwpy/cli/cliproduct.py]: @areeda, if the user gives multiple arguments to `--title`, doesn't this mean that only the last given `title_line` will be preserved?

File: gwpy/cli/cliproduct.py [hunk 2]
```diff
@@ -695,9 +697,14 @@ def run(self):
...
 
         self.log(3, 'Arguments:')
+        hastex = False
         argsd = vars(self.args)
         for key in sorted(argsd):
             self.log(3, '{0:>15s} = {1}'.format(key, argsd[key]))
+            if isinstance(argsd[key], str):
+                hastex |= '$' in argsd[key]
+        hastex |= self.args.mode == 'spectrum'
+        rc('text', usetex=hastex)
```
Comment (duncanmmacleod) [inline gwpy/cli/cliproduct.py]: @areeda, I'm not sure this is the right way to go, given that matplotlib has (some) support for mathtext without using latex. Can you experiment with always using functions like `label_to_latex` but _not_ using `text.usetex`?

File: gwpy/cli/cliproduct.py (hunk 1 above)
Comment (duncanmmacleod) [reply gwpy/cli/cliproduct.py]: @areeda, ping?

**Comment by areeda** [comment]
Withdrawn #977 now resolves issue.

**Comment by duncanmmacleod** [comment]
@areeda, #977 does not fix the `title` issues that this PR address. Can you rebase this PR to only implement those fixes and reopen, or post a new PR?

**Pull request closed without merging**